posthog/api/test/batch_exports/test_update.py

```diff
@@ -511,3 +511,66 @@ def test_switching_snowflake_auth_type_to_keypair_requires_private_key(client: H
         batch_export = get_batch_export_ok(client, team.pk, batch_export["id"])
         assert batch_export["destination"]["type"] == "Snowflake"
         assert batch_export["destination"]["config"]["authentication_type"] == "password"
+
+
+def test_can_validate_snowflake_destination_with_patched_config(client: HttpClient, temporal):
```

Comment:
I think we have a similar test above for patching Snowflake destination config?

Reply:
Similar, but this one maintains the authentication type instead of changing it. I can still delete it though, the other test was the one that caught the problem in the end.

Reply:
What do you think?

Reply:
I've dropped it :+1: